packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -3070,7 +3135,7 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
     });
   }
 
-  static void setupDefaultPubRootDirectory(TestWidgetInspectorService service) {
+  static String generateTestPubRootDirectory2(TestWidgetInspectorService service){
```

Comment:
I assume adding a 2 to this method name was an accident.

Reply:
also missing a space between `)` and `{`

Reply:
yes ty, i put it there while moving the function around